common/types.go

```diff
@@ -326,25 +326,93 @@ func (b *BidTraceV2WithTimestampJSON) ToCSVRecord() []string {
 	}
 }
 
+type BidTraceV2WithBlobFields struct {
+	builderApiV1.BidTrace
```

Comment:
could extend V2

Reply:
This would expose the blob data to the data api as it uses the same json structs